ucx.spec.in

```diff
@@ -301,6 +301,8 @@ process to map the memory of another process into its virtual address space.
 
 
 %changelog
+* Mon Feb 10 2020 Yossi Itigin <yosefe@mellanox.com> 1.10.0-1
```

Comment:
You probably want to bump the date ?

Reply:
fixed + added test

Reply:
Does rpm build complains about duplicated entries ?

Reply:
no